lighthouse-core/test/fixtures/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "commonjs",
+  "//": "All fixtures are commonjs until core/ is modules."
```

Comment:
yikes, this directory shows some of the trickiness in store for us for `core/` :/